<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOptionsWidget</class>
 <widget class="QWidget" name="GeneralOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="showCurosrLinesCheckBox">
       <property name="text">
        <string>Show lines under cursor</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showRulersCheckBox">
       <property name="text">
        <string>Show rulers</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Default origin position:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="originTypeComboBox">
         <item>
          <property name="text">
           <string>Workspace center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TopLeft workspace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BottomLeft workspace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TopRight workspace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BottomRight workspace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Typon center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TopLeft typon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BottomLet typon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TopRight Typon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BottomRight typon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="originGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>x:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYUnitsSpinBox" name="xOriginSpinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MYUnitsSpinBox" name="yOriginSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Default unit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MYUnitsComboBox" name="unitsComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Default typon size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MYUnitsSpinBox" name="typonWidthSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="value">
          <double>160.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MYUnitsSpinBox" name="typonHeightSpinBox">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Copper thickness:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="copperThicknessSpinBox">
         <property name="suffix">
          <string>µm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Temp rise:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tempRiseSpinBox">
         <property name="suffix">
          <string>°C</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="saveWorkSpaceAutCheckBox">
       <property name="text">
        <string>Automatically save workspace on quit</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="zoomOnWheelCheckBox">
       <property name="text">
        <string>Zoom on mouse wheel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trackOpacityCheckBox">
       <property name="text">
        <string>Show semi transparent tracks</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYUnitsSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>utils/myunitsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>MYUnitsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">utils/myunitscombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
